### ISSUE TITLE
BUG https://cdn.skypack.dev/three@0.136.2 not found

### ISSUE THREAD

**Issue by pflagerd** [looeee/discoverthreejs-site#92]
Please report bugs, rendering errors, CSS problems, cross-browser issues and so on here. 
If you have found a mistake in the text, you can also report it here, although there's no need to report OS/browser details in that case. 

**Describe the bug**
When performing "Download as Zip" from IDE, zipped site contains references to https://cdn.skynet.dev/three@0.132.2 which does not exist.  As of this date,  references to https://cdn.skynet.dev/three produce 0.148.0 as the only option:

```
/*
 * Skypack CDN - three@0.148.0
 *
 * Learn more:
 *   📙 Package Documentation: https://www.skypack.dev/view/three
 *   📘 Skypack Documentation: https://www.skypack.dev/docs
 *
 * Pinned URL: (Optimized for Production)
 *   ▶️ Normal: https://cdn.skypack.dev/pin/three@v0.148.0-ZdnPTf2EskNtHkVhjjpp/mode=imports/optimized/three.js
 *   ⏩ Minified: https://cdn.skypack.dev/pin/three@v0.148.0-ZdnPTf2EskNtHkVhjjpp/mode=imports,min/optimized/three.js
 *
 */
```

// Browser-Optimized Imports (Don't directly import the URLs below in your application!)
export * from '/-/three@v0.148.0-ZdnPTf2EskNtHkVhjjpp/dist=es2019,mode=imports/optimized/three.js';
export {default} from '/-/three@v0.148.0-ZdnPTf2EskNtHkVhjjpp/dist=es2019,mode=imports/optimized/three.js';

**To Reproduce**
1. From a browser, navigate to https://discoverthreejs.com/book/first-steps/first-scene/
2. Click on the "Show IDE" icon in the far upper-left corner of the browser client area to make sure the IDE is visible:
![image](https://user-images.githubusercontent.com/246217/212504385-41ac5fe5-74ae-419b-a16c-b24d0876c85e.png)
3. From the IDE, click the "Download as Zip" icon:
![image](https://user-images.githubusercontent.com/246217/212504424-7d8f3524-dfb6-49fa-a549-caab3880a18c.png)
4. When the download completes, you should have a file like the following in your Downloads/ directory:
![image](https://user-images.githubusercontent.com/246217/212504455-34894dd8-bf6e-40bb-947b-61e63e17c2e4.png)
5. Unzip the file and locate src/main.js. Observe that it contains an import statement as follows.
```
import {
  BoxBufferGeometry,
  Color,
  Mesh,
  MeshBasicMaterial,
  PerspectiveCamera,
  Scene,
  WebGLRenderer,
} from 'https://cdn.skypack.dev/three@0.136.2';
```
6. Observe that the import refers to three version 0.136.2.
7. From a browser, attempt to navigate to 'https://cdn.skypack.dev/three@0.136.2'
8.  Observe this message in your browser: `Package "three" exists, but could not match version "0.136.2".`

**Expected behavior (optional)**
Attempting to navigate to 'https://cdn.skypack.dev/three@0.136.2' should result in something like this in your browser:

![image](https://user-images.githubusercontent.com/246217/212504545-c40529be-918e-472c-85a7-ff74b9d1c195.png)

**Desktop (please complete the following information):**
 - OS:  Ubuntu Linux   22.04.1 LTS
 - Browser: Chrome
 - Version: 109.0.5414.74 (Official Build) (64-bit)

**Additional context**

**Comment by gisfinder** [comment]
the source file ["vendor/three/build/three.module.js" ](https://discoverthreejs.com/book/introduction/get-threejs/) is wrong.

**Comment by pflagerd** [comment]
I agree that the source file "vendor/three/build/three.module.js" looks wrong.  It looks like html.  But I cannot find where it is referenced in the source code anywhere.  In fact, after finding a fix for the cdn.skypack.dev version problem, I removed the vendor/ directory entirely and suffered no ill effects.  Code appeared to work the same.

**Comment by pflagerd** [comment]
I found that if I replaced 0.136.2 with 0.149.0, I could run MOST of the code examples.  However, I also found a compatibility problem with the import in the controls.js file of those code examples that have controls.js (not all do).

```
import { OrbitControls} from 'https://cdn.skypack.dev/three@0.149.0/examples/jsm/controls/OrbitControls.js';
```

causes a Chrome/Chromium 118.0.5993.88 browser instance to complain in its console (and no scene is displayed) thus:

```
Uncaught TypeError: Failed to resolve module specifier "three". Relative references must start with either "/", "./", or "../".
```

Seems that the problem is somewhat unique to cdn.skypack.dev as replacing it with esm.sh or jspm.dev in the offending `import` results in the scene being correctly displayed.

e.g.
```
import { OrbitControls} from 'https://esm.sh/three@0.149.0/examples/jsm/controls/OrbitControls.js';
```

**Comment by pflagerd** [comment]
I guess I will work on replacing cdn.skypack.dev and changing the version number to 0.149.0 in my fork.

It would probably be wise to make it clear in the text of the site and/or the code examples themselves  that we are using 0.149.0 which is not the most recent version.

**Comment by pflagerd** [comment]
Also begs the question "How about examples which use the npm import style for running locally?"
